open-machine-learning-jupyter-book/assignments/ml-fundamentals/ML-linear-regression.ipynb

```diff
@@ -0,0 +1,1082 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    
</pre><p>remove the unused empty line</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open-academy/machine-learning/pull/187/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='85'/>